src/kbc_modules/cc_kbc/crypto/mod.rs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2022 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use aes_gcm::aead::{Aead, NewAead};
+use aes_gcm::Aes256Gcm;
+use anyhow::*;
+use rsa::pkcs8::EncodePublicKey;
+use rsa::{PaddingScheme, RsaPrivateKey, RsaPublicKey};
+use serde::{Deserialize, Serialize};
+use sha2::{Digest, Sha384};
+
+const RSA_ALGORITHM: &str = "rsa-pkcs1v15";
+const RSA_PUBKEY_LENGTH: usize = 2048;
+const NEW_PADDING: fn() -> PaddingScheme = PaddingScheme::new_pkcs1v15_encrypt;
+
+const AES_GCM_256_ALGORITHM: &str = "aes-gcm-256";
+
+// The key inside TEE to decrypt confidential data.
+#[derive(Debug, Clone)]
+pub struct TeeKey {
+    private_key: RsaPrivateKey,
+    public_key: RsaPublicKey,
+}
+
+// The public key of TEE.
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct TeePubKey {
```

Comment:
So this is just the struct that we use to export the pubkey, right? It might be worth mentioning that in the comment since we have another (different) pubkey below.